src/nyc_trees/apps/survey/models.py

```diff
@@ -40,6 +41,11 @@ class Survey(NycModel, models.Model):
                                  related_name='surveys_as_teammate',
                                  blank=True)
     is_flagged = models.BooleanField(default=False)
+    has_trees = models.BooleanField(default=True)
+    is_mapped_in_blockface_polyline_direction = models.BooleanField(
+        default=True)
+    is_left_side = models.BooleanField(default=True)
```

Comment:
That was my inclination too, but Django's default is `None` and I didn't want any `None` values in the DB. That said, It will take explicit validation code to catch omissions either way, and it's better to test for a value of `None` in the `clean` method then to test for the presence of fields in `submit_survey`, so I'll change it.